CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 option(ENABLE_GUI "Build with GUI" ON)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
-set(CMAKE_CXX_FLAGS_DEBUG "-g")
+set(CMAKE_CXX_FLAGS_DEBUG "-g -O3")
```

Comment:
Probably remove again or reduce to O1 or O2: variables can be optimized away, which makes debugging annoying